rusoto/core/src/signature.rs

```diff
@@ -6,6 +6,8 @@
 //! If needed, the request will be re-issued to a temporary redirect endpoint.  This can happen with
 //! newly created S3 buckets not in us-standard/us-east-1.
 
+extern crate hex;
```

Comment:
It looks like we already have `extern crate hex` in `lib.rs`. Should this be a `use hex` instead?

Reply:
Thanks, not sure why I swapped to that format. Fixed now!